WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -362,12 +391,20 @@ public static boolean hasPendingOrInProgressMediaUploadsForPost(PostModel postMo
         return postModel != null && MediaUploadHandler.hasPendingOrInProgressMediaUploadsForPost(postModel);
     }
 
+    public static boolean hasAnyFailedMediaUploadsForPost(PostModel postModel) {
```

Comment:
Removed the method altogether in 2c577cb. As of the change requested in https://github.com/wordpress-mobile/WordPress-Android/pull/6366/files/7194bbd103ac9e944350af320ea82911152633ab#r129034268 the method was left unused.